CONTRIBUTING.md

```diff
@@ -30,3 +30,66 @@ Use culturally neutral language:
 
   What you may consider well-known may be entirely obscure and distracting to people from different backgrounds.
 
+# Target audience
+
+The main target audience are software developers with basic experience on the Linux command line.
+Specific prerequisite knowledge and skills should always be stated, see [Structure](#structure) for details.
+
+These guides should work for autodidacts who do not have access to hand-holding or personal training, nor possibility for learning through osmosis.
+
+# Structure
+
+Each guide should have the following structure:
+
+- What will you learn? (problem statement and learning goals)
+- What do you need? (prerequisites)
+
+  The material should always be written such that the stated prerequisites are sufficient to achieve learning goals.
+
+  - previous chapters
+  - domain-specific skills or knowledge
+
+- How long does it take? (reading time)
+
+  This will depend on the learner's pre-existing knowledge and proficiency.
+  Note here how optional skills or knowledge may influence reading time.
```

Comment:
How should the reading time be estimated then? Do we give multiple times per people profile ?

Reply:
Yes, that's what I attempted here: https://github.com/NixOS/nix.dev/pull/267/files#diff-c08ceee24137f72990af94b2e6a5eb487ec1ec767437a6082ba06d9d4244b835R64